src/send/saga.ts

```diff
@@ -303,14 +287,72 @@ export function* sendPaymentSaga({
   }
 }
 
+export function* sendTransactionsSaga(
+  serializablePreparedTransactions: SerializableTransactionRequest[],
+  networkId: NetworkId,
+  createBaseStandbyTransactions: ((
+    hash: string,
+    feeCurrencyId?: string
+  ) => BaseStandbyTransaction)[]
+) {
```

Comment:
Great!

Should we maybe document the arguments?
Explaining what `createBaseStandbyTransactions` is for.

Also what do you think of moving this saga to `src/viem/sagas`?
And maybe renaming it to `sendPreparedTransactions`?

Reply:
yes good ideas! i renamed and moved to the viem folder. also added some jsdocs for the saga input params!